src/types/interfaces.ts

```diff
@@ -25,4 +37,29 @@ export interface CurrentLocationData {
     location_type: string,
     title: string,
     woeid: number,
-}
\ No newline at end of file
+}
+
+export interface BadgeProps {
+    icon: string,
+    alt: string,
+    className: string,
+    onClick: MouseEventHandler
+}
+
+export interface ButtonProps {
```

Comment:
leave props interfaces inside of component or in separate file in the same folder